src/Mono.Android/Java.Lang/Object.cs

```diff
@@ -16,10 +16,8 @@ public partial class Object : IDisposable, IJavaObject, IJavaObjectEx
 #endif  // JAVA_INTEROP
 	{
 		IntPtr key_handle;
-		IntPtr weak_handle;
```

Comment:
These fields can't be removed, as they're used from `monodroid-glue.cc`.  Ditto in `Throwable.cs`.